lib/rspec/support/differ.rb

```diff
@@ -7,6 +7,7 @@ module RSpec
   module Support
     # rubocop:disable ClassLength
     class Differ
+      LINE_BREAK = "\n"
```

Comment:
Well, as I commented in https://github.com/bf4/rspec-support/commit/ce333af71a110fe5f080b3b0e8cc37c8cb05db6a I was thinking of using `LINE_BREAK` as a way to get the `__ENCODING__` of strings generated in `differ.rb`, that also happened to be a nice cleanup of string generation. Turns out it didn't work like I thought it did, so, the only reason to keep it now is cleanup, if you like. I'm going to try again with a less aggressive strategy.